exercises/rational-numbers/src/example.c

```diff
@@ -0,0 +1,125 @@
+#include "rational_numbers.h"
+#include <math.h>
+
+static int16_t absolute_int(int16_t n)
+{
+   const int16_t mask = n >> 15;
+   return (n + mask) ^ mask;
+}
```

Comment:
Nit: Shouldn't this instead be the `abs()` library function?

Reply:
😆 😆  I can't believe I did this!